<launch>
	<node  pkg="model_completion" type="in_hand_object_modeling" name="get_mask_client" output="screen"  >
	<param name="width" value="640" />
	<param name="height" value="480" />
	<param name="fovy" value="47" />
	<param name="camera_frame" value="openni_rgb_optical_frame" />
	<param name="image_topic" value="/camera/rgb/image_mono" />
	<!-- <param name="camera_frame" value="narrow_stereo_optical_frame" /> -->
	<!-- <param name="image_topic" value="/narrow_stereo/left/image_rect" /> -->
	<param name="save_images" value="true" />
	<param name="mode" value="knn_search" />
	<param name="tf_topic" value="/tf" />
	<param name="min_nn" value="4" />	
    <param name="knn" value="4" />
  
	</node>
	<param name="robot_description" textfile="$(find model_completion)/robot/robot.xml"/>
	<node  pkg="robot_mask" type="robot_mask" name="robot_mask" output="screen" >
	</node>
</launch>

<!-- launch-prefix="xterm -e gdb -args" -->
